spec/models/timecard_spec.rb

```diff
@@ -19,60 +55,182 @@
+        expect(described_class.most_recent.count).to eq(3)
+      end
+
+      it 'returns the most recent timecard as its first element' do
+        expect(described_class.most_recent.first).to eq(@first_timecard)
+      end
+
+      it 'returns the oldest timecard as its last element' do
+        expect(described_class.most_recent.last).to eq(@last_timecard)
+      end
+    end
+  end
+
+  describe 'working scope' do
+    it 'returns a chainable relation' do
+      expect(described_class.working).to be_a_kind_of(ActiveRecord::Relation)
+    end
+
+    context 'given 3 timecards: an Incomplete one with no end_time; another also Incomplete 
+             but with an end_time; a generic timecard' do
+      before(:example) do
+        @working_timecard  = create(:timecard, person: @cj, start_time: Time.current,
+                                    end_time: nil, status: 'Incomplete')
+        @finished_timecard = create(:timecard, person: @cj, start_time: 1.hour.ago,
+                                    end_time: Time.current, status: 'Incomplete')
+        @timecard          = create(:timecard, person: @cj)
+      end
+
+      it 'returns the timecard with Incomplete status and no end_time' do
+        expect(described_class.working).to contain_exactly(@working_timecard)
+      end
+    end
+  end
+
+  describe 'active scope' do
+    it 'returns a chainable relation' do
+      expect(described_class.active).to be_a_kind_of(ActiveRecord::Relation)
+    end
+    
     it 'returns all timecards that are not cancelled' do
       incomplete_timecard = create(:timecard, person: @cj, status: 'Incomplete')
       unverified_timecard = create(:timecard, person: @cj, status: 'Unverified')
       error_timecard = create(:timecard, person: @cj, status: 'Error')
       verified_timecard = create(:timecard, person: @cj, status: 'Verified')
       cancelled_timecard = create(:timecard, person: @cj, status: 'Cancelled')
       active_timecards = Timecard.active
-      expect(active_timecards).to include(incomplete_timecard)
-      expect(active_timecards).to include(unverified_timecard)
-      expect(active_timecards).to include(error_timecard)
-      expect(active_timecards).to include(verified_timecard)
+      expect(active_timecards).to include(incomplete_timecard, unverified_timecard, error_timecard, verified_timecard)
       expect(active_timecards).not_to include(cancelled_timecard)
     end
   end
 
-  describe "creation" do
-    it "has a valid factory" do
-      timecard = build(:timecard, person: @cj)
-      expect(timecard).to be_valid
+  describe 'older_than scope' do
+    it 'returns a chainable relation' do
+      expect(described_class.older_than(1)).to be_a_kind_of(ActiveRecord::Relation)
     end
+    
+    context 'given 2 timecards with different start_times' do
+      before(:example) do
+        @timecard_with_7hours = create(:timecard, person: @cj,
+          start_time: 7.hours.ago, end_time: 1.hour.ago)
```

Comment:
I really like all the tests!
But wouldn't this be a person with six hours ? ( 7-1)